ibc-core/ics24-host/types/expanded.rs

```diff
@@ -0,0 +1,3890 @@
+#![feature(prelude_import)]
```

Comment:
This `expanded.rs` file looks irrelevant and can be removed.

Reply:
[e92fa78](https://github.com/cosmos/ibc-rs/pull/1013/commits/e92fa7820982af117d90d25c698282868cf0d507)